server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/jetty/CBJettyServer.java

```diff
@@ -179,6 +180,16 @@ public int getRefreshCookieAge() {
                 return 1;
             }
         }*/;
+        var maxIdleSeconds = CBPlatform.getInstance().getSessionManager().getMaxSessionIdleTime();
```

Comment:
Add `CBApplication application` to method arguments and get session manager from it

Reply:
Done